.gitignore

```diff
@@ -201,5 +201,8 @@ typings/
 schema-directives.graphql
 _ide_helper.php
 
+# Algolia
+config/scout.php
```

Comment:
Som jeg nævner i selve filen scout.php - så forslår jeg at du dropper at "spærre" for denne fil - og bruger .env til at gemme på secrets. Vi har brug for config-filen til at starte scout op :)

Reply:
lol. Den er revertet i næste commit